tools/get_config.py

```diff
@@ -48,12 +48,12 @@
 
     # Target
     if options.mcu is None :
-        args_error(parser, "[ERROR] You should specify an MCU")
+        args_error(parser, "argument -m/--mcu is required")
```

Comment:
Why did you remove ``[ERROR]`` ? wasn't that an intention warning/error/etc ? Does this PR clean all [ERROR] messages ?